internal/base_component.go

```diff
@@ -307,3 +308,10 @@ func (base *BaseComponent) InvalidateObject(name string) {
 		base.next.InvalidateObject(name)
 	}
 }
+
+func (base *BaseComponent) GetFileBlockOffsets(options GetFileBlockOffsetsOptions) (*common.BlockOffsetList, error) {
```

Comment:
We need to add this method prototype to component interface. BaseComponent is just implementation of that interface.

Reply:
Will add to the component interface.